app.js

```diff
@@ -0,0 +1,48 @@
+import express from 'express';
+import cors from 'cors';
+import morgan from 'morgan';
+import methodOverride from 'method-override';
+import { joiErrors } from './middlewares';
+import routes from './routes';
+
+const isProduction = process.env.NODE_ENV === 'production';
+
+// Create global app object
```

Comment:
Let's aim to reduce the comments. They end up being too many within a project. Some comments are necessary some are not.

Reply:
I will mark which ones are necessary. Please delete all others including this one.